src/test/features/generateReport.feature

```diff
@@ -0,0 +1,20 @@
+@report
+Feature: Report
+
+Background: 
+ Given User passes the authorization
+
+ Scenario: Generate a new report
+        Given User is in landing page
+        When User logins as a "admin"
+        Then User is in Dashboard page
+        When User navigates to Reports tab
```

Comment:
One step definition should be responsible for navigating user to ANY given page.